src/WebApi/DependencyInjection.cs

```diff
@@ -1,18 +1,50 @@
-﻿using SSW.CleanArchitecture.Application.Common.Interfaces;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SSW.CleanArchitecture.Application.Common.Interfaces;
 using SSW.CleanArchitecture.Infrastructure.Persistence;
+using SSW.CleanArchitecture.WebApi.HealthChecks.EntityFrameworkDbContextHealthCheck;
 using SSW.CleanArchitecture.WebApi.Services;
 
 namespace SSW.CleanArchitecture.WebApi;
 
 public static class DependencyInjection
 {
-    public static void AddWebApi(this IServiceCollection services)
+    public static void AddWebApi(this IServiceCollection services, IConfiguration config)
     {
         services.AddHttpContextAccessor();
         services.AddScoped<ICurrentUserService, CurrentUserService>();
 
         services.AddHealthChecks()
```

Comment:
Should all this be in an AddHealthChecks method instead of the AddWebApi method?

Reply:
Good suggestion.  That would tidy it up a lot.

Reply:
As in a whole separate extension method just for HealthChecks?

Reply:
I'm happy for the code to be hooked in via `AddWebApi()`, but perhaps wrap up all HealthCheck related code into it's own extensions method that is called from `AddWebApi()`.

Reply:
Done